app/src/main.rs

```diff
@@ -118,71 +100,67 @@ fn main() {
 
     let mut retval = sgx_status_t::SGX_SUCCESS;
 
-    let binary_median_int = fs::read(WASM_FILE_MEDIAN_INT).unwrap();
-    let binary_median_float = fs::read(WASM_FILE_MEDIAN_FLOAT).unwrap();
-
-    let binary_mean_int = fs::read(WASM_FILE_MEAN_INT).unwrap();
-    let binary_mean_float = fs::read(WASM_FILE_MEAN_FLOAT).unwrap();
-
-    let binary_sd_int = fs::read(WASM_FILE_SD_INT).unwrap();
-    let binary_sd_float = fs::read(WASM_FILE_SD_FLOAT).unwrap();
-
-    let mut result_out_median_int = 0i32;
-    let mut result_out_median_float = 0f32;
-
-    let mut result_out_mean_int = 0i32;
-    let mut result_out_mean_float = 0f32;
-
-    let mut result_out_sd_int = 0f32;
-    let mut result_out_sd_float = 0f32;
+    //// Mean test
+    // Numbers float and integer mixed
+     let data_median = read_data_from_json(
+        "/root/workspace/wasm-exec-sgx/get-median-wasm/test_data.json",
```

Comment:
The file path needs to change, "/home/bill/workspace..." worked for me for my system.